frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/map-image-popup/map-image-popup.component.ts

```diff
@@ -52,4 +52,12 @@ export class MapImagePopupComponent implements PopupComponent, OnInit {
             newHeight,
         });
     }
+
+    public lock(newLocked: boolean) {
```

Comment:
I think `lock` is okay, but I would propose two different Action Reducers, one doing `Lock` and one doing `Unlock` instead of a `Lock` action that can also unlock or call this action something like `Toggle lock` (but the problem there is you can't enforce it to be locked, when you don't know the current state, don't know what happens if something gets out of sync or if this out of sync would even be a possibility).

Reply:
To avoid having two actions for this and the trouble a `toggle`-action might have, we've decided to use `Set Locked` as the name for the action ([1611dfa](https://github.com/hpi-sam/digital-fuesim-manv/pull/545/commits/1611dfa760d14b6c98785c5169c1e58f02b05e58)). Does this look good to you?

Reply:
What about something like `Set isLocked` (the variable name) or `Set LockedStatus` or `LockStatus`?

Reply:
> > To avoid having two actions for this and the trouble a `toggle`-action might have, we've decided to use `Set Locked` as the name for the action ([1611dfa](https://github.com/hpi-sam/digital-fuesim-manv/pull/545/commits/1611dfa760d14b6c98785c5169c1e58f02b05e58)). Does this look good to you?
> 
> I'm fine with it.
> 
> I assume @anonym-HPI wanted two separate actions because of [this](https://redux.js.org/style-guide/#model-actions-as-events-not-setters) best practice?
> Because this action only sets the `lock` property and is completely independent of other actions, I think your setter approach is better.
> 
> This should be refactored to `lock` and `unlock` actions, though, as soon as the reducers do more.

Didn't knew this style guide, but I am fine with one action, but I would change the action name (see my previous comment).